### ISSUE TITLE
Android browser passing in invalid `androidbrowser`

### ISSUE THREAD

**Issue by linnea** [DevExpress/testcafe-browser-provider-browserstack#165]
We were getting intermittently failing tests when using the 'browserstack:Samsung Galaxy S20@10.0', android browser.

```
ERROR Cannot establish one or more browser connections.
5 of 10 browser connections have not been established:
- browserstack:Samsung Galaxy S20@10.0
- browserstack:Samsung Galaxy S20@10.0
- browserstack:Samsung Galaxy S20@10.0
- browserstack:Samsung Galaxy S20@10.0
- browserstack:Samsung Galaxy S20@10.0

Hints:
- The host machine may not be powerful enough to handle the specified concurrency factor (5). Try to decrease the concurrency factor or allocate more computing resources to the host machine.
- Increase the value of the "browserInitTimeout" option if it is too low (currently: 2 minutes for all browsers). This option determines how long TestCafe waits for browsers to be ready.
- The error can also be caused by network issues or remote device failure. Make sure that your network connection is stable and you can reach the remote device.
```

We tried a lot of different things, and reached out to Browserstack support for help. They identified that the browser being passed in was invalid for Samsung type devices:

<img width="524" alt="Screen Shot 2022-04-14 at 11 21 41 AM" src="https://user-images.githubusercontent.com/8867604/163465736-73c1e52f-e34a-43d6-8327-5f29591b4f0c.png">
<img width="1121" alt="Screen Shot 2022-04-14 at 11 23 07 AM" src="https://user-images.githubusercontent.com/8867604/163465743-0d516cc9-7965-4a3a-b81d-8897b304ce11.png">
 
"androidbrowser is not an accepted parameter for browsers on android devices. You should choose either a chrome or samsung browser using the capabilities set forth in our capabilities generator"

"Looking into the output capabilities on my end, for Samsung devices the browser capability is showing up as null vs Samsung/Chrome. This may explain why the tests aren't connecting to a browser. Are you seeing this in non-Samsung devices as well?"

As soon as we updated our Android device to a Google Pixel, we saw a much lower frequency of tests failing. However, they still seem to fail. 

```
 "testcafe": "^1.18.5",
    "testcafe-browser-provider-browserstack": "^1.13.2-alpha.1”, (have been using this version for > 14 months, intermittent issues started happening a few months ago)
    "testcafe-reporter-junit": "^3.0.2",
    "testcafe-vue-selectors": "^3.1.1”,
```

testcaferc.js
```
module.exports = {
	src: './tests/Javascript/prime/e2e/fixtures',
	assertionTimeout: 15000,
	selectorTimeout: 15000,
	pageLoadTimeout: 20000,
	quarantineMode: {
		successThreshold: 3,
		attemptLimit: 5,
	},
	concurrency: 5,

	hostname: 'localhost',

	reporter: [
		{
			name: 'spec',
		},
		{
			name: 'junit',
			output: 'testcafe-results.xml',
		},
	],
};
```

.testcaferc.browserstack.js
```
const dotenv = require('dotenv');
const config = require('./.testcaferc.js');

dotenv.config();

module.exports = Object.assign(config, {
	browserInitTimeout: 120000, // 2 minute timeout
	browsers: [
		'browserstack:chrome@98.0:Windows 10',
		'browserstack:Samsung Galaxy S20@10.0',
		// 'browserstack:iPhone X@11.0',
	],
});
```

Until we switched our 
		'browserstack:Samsung Galaxy S20@10.0',

to
		'browserstack:Google Pixel 5@11.0',

However, we're still getting failures and wondering if the android browser param can be updated to `chrome` instead of `androidbrowser` Is there a way we can designate this with the browser type? I.e. `browserstack:chrome:Samsung Galaxy S20@10.0` instead of just `browserstack:Samsung Galaxy S20@10.0`

**Comment by felis2803** [comment]
Hello @linna,

Browserstack support has taken you down the wrong path:
1. `androidbrowser` is the default browser defined by Browsersack itself. testcafe-browser-provider-browserstack does not specify this name anywhere. You can verify this by searching for the word `androidbrowser` in this repository.
2. Tests pass successfully with this browser name:
![изображение](https://user-images.githubusercontent.com/34184692/165099046-5a6069c3-6cad-4f41-88fa-deb811031eb4.png)


However, I do see instability when testing in parallel with more than 5 threads. I can't tell yet if the cause of the problem is on the TestCafe or Browserstack side. I created a separate issue to investigate this problem: https://github.com/DevExpress/testcafe-browser-provider-browserstack/issues/166. Stay tuned for updates on it.
